code/game/machinery/dance_machine.dm

```diff
@@ -322,14 +322,12 @@
 
 /obj/machinery/disco/proc/dance(mob/living/M) //Show your moves
 	set waitfor = FALSE
-	switch(rand(0,9))
+	switch(rand(0,6))
 		if(0 to 1)
 			dance2(M)
 		if(2 to 3)
 			dance3(M)
 		if(4 to 6)
-			dance4(M)
-		if(7 to 9)
 			dance5(M)
```

Comment:
If you're going to do this, I'd suggest renaming `dance5` to `dance4` to keep things consistent, it's probably just me though